### ISSUE TITLE
better error when trying to run test to a 3.5 host

### ISSUE THREAD

**Issue by bltierney** [perfsonar/pscheduler#392]
The error below is for a test from a 4.0 host to a 3.5 host, and will confuse users.

How hard would it be to figure out / infer its a 3.5 host, and just print:
"Error, remote endpoint is not running perfSONAR 4.0. Try running bwctl instead."

Or something like that...

 pscheduler task --tool nuttcp throughput --interval 1 --dest slac-pt1.es.net
Submitting task...
Task URL:
https://nettest.lbl.gov/pscheduler/tasks/88ee467d-f254-4b9e-addd-f44f8e26552e
Running with tool 'nuttcp'
Fetching first run...

Next scheduled run:
https://nettest.lbl.gov/pscheduler/tasks/88ee467d-f254-4b9e-addd-f44f8e26552e/runs/3b0a13f2-d343-45e6-a1f8-d54b32a2a46b
Starts 2017-03-31T09:33:37-07:00 (~61 seconds)
Ends   2017-03-31T09:33:53-07:00 (~15 seconds)
Waiting for result...
Run failed: nuttcp requires exactly 2 participants, got 1

Please note that if you are testing with a mixture of pScheduler enabled hosts and legacy BWCTL hosts, certain options such as --tool may result in this error message. To ensure backwards compatibility mode is used, do not specify --tool if either end is a BWCTL based host. If possible, the perfSONAR team strongly recommends upgrading as backwards compatibility support will be dropped in a future release.
+ pscheduler task --tool nuttcp throughput --omit 5 --duration PT20S --dest slac-pt1.es.net
Submitting task...
Unable to post task: POST https://nettest.lbl.gov/pscheduler/tasks returned 422: Unable to complete request: Couldn't find a tool in common among the participants.

**Comment by mfeit-internet2** [comment]
slac-pt1 was upgraded to 4.0rc3, but I was able to duplicate this testing to ps-deb2.es.net.  Here's what I found:

Restricting the test to the `nuttcp` tool won't work against a 3.5 host because it doesn't do the `--omit` switch.  That's not the right tool to be using anyway since it was designed to be used with pScheduler on both ends.

I may be missing something here (but I don't think so), but we have no tool capable of invoking `bwctl` in a way that will use `nuttcp`.  We need a `bwctlnuttcp` tool to go alongside `bwctliperf2` and `bwctliperf3` we already have.  I believe adding one with a high preference value like we did for bwctliperf3 will solve the problem.

This should be an easy effort since it should, in theory, be nearly a clone of `bwctliperf*`.
